File: SWIFT/Cap04/c04ex03.swift
// c04ex03.swift

import Foundation

var A, B, X, R: Int

print("Entre o 1o. valor numerico: ", terminator: "")
if let inputA = readLine(), let intA = Int(inputA) {
    A = intA
} else {
    fatalError("Entrada invalida.")
}

print("Entre o 2o. valor numerico: ", terminator: "")
if let inputB = readLine(), let intB = Int(inputB) {
    B = intB
} else {
    fatalError("Entrada invalida.")
}

X = A + B

if (X >= 10) {
    R = X + 5
} else {
    R = X - 7
}

print("Resultado = \(R)")

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: KOTLIN/Cap04/c04ex03.kt
// c04ex03.kt

fun main() {
    var A: Int
    var B: Int
    var X: Int
    var R: Int

    print("Entre o 1o. valor numerico: ")
    A = readLine()?.toInt() ?: 0

    print("Entre o 2o. valor numerico: ")
    B = readLine()?.toInt() ?: 0

    X = A + B

    if (X >= 10)
        R = X + 5
    else
        R = X - 7

    println("Resultado = $R")

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
